code/game/turfs/simulated/floor.dm

```diff
@@ -213,6 +213,9 @@ GLOBAL_LIST_INIT(icons_to_ignore_at_floor_init, list("damaged1","damaged2","dama
 	var/obj/item/thing = user.get_inactive_hand()
 	if(!thing || !prying_tool_list.Find(thing.tool_behaviour))
 		return
+	if(istype(src, /turf/simulated/floor/transparent/glass))
+		to_chat(user, "<span class='danger'>You need to hold two sheets of metal to dismantle \the [src]!</span>")
+		return
```

Comment:
This should go in an override at the transparent glass floor level. We always want to avoid `istype(src, ...)`